TicTacToeApp/ComputerPlayer.swift

```diff
@@ -7,6 +7,16 @@ public class ComputerPlayer {
   }
 
   public func makeMove(gameState: Game) -> Int {
+    var move: Int?
+    let serviceUrl = "http://107.170.25.194:5000/"
```

Comment:
I would make the index of the server be instructions on how to use the API, with the actual call to the computer API be something like `"http://107.170.25.194:5000/get_move"`.